app/src/main/java/presenter/GithubUsersPresenter.java

```diff
@@ -0,0 +1,54 @@
+package presenter;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.util.List;
+
+import model.GithubUsers;
+import model.GithubUsersResponse;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import service.GithubService;
+import view.GithubUsersView;
+
+public class GithubUsersPresenter {
+    private static final String TAG = "GithubUsersPresenter";
```

Comment:
Could you please put this string variable in the String resource file such that you just reference it here